advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -61,8 +62,8 @@ class TodoApp extends Component {
 	}
 
 	handleOnChange(e) {
-		let value = e.target.value
-		if(!isUndefined(value) && !isEmpty(value)){
+		var value = e.target.value

```

Comment:
This variable is declared with `var` but would be better suited as a `const` since it is never reassigned.